librz/core/tui/panels.c

```diff
@@ -1147,10 +1147,14 @@ char *__find_cmd_str_cache(RzCore *core, RzPanel *panel) {
 
 char *__apply_filter_cmd(RzCore *core, RzPanel *panel) {
 	char *out = malloc(strlen(panel->model->cmd) + 1024);
-	if (!out || !panel->model->filter) {
+	if (!out) {
+		RZ_LOG_ERROR("Fail to allocate the memory\n");
 		return out;
 	}
 	strcpy(out, panel->model->cmd);
+	if (!panel->model->filter) {
+		return out;
+	}
```

Comment:
Is this really necessary? I expect that when `filter` is NULL also `n_filter == 0` so the following for-loop would not be executed anyway.